aws-config/defaults.json

```diff
@@ -1,9 +1,11 @@
 {
-  "availability_zone": "us-east-1e",
+  "availability_zone": "us-east-1c",
   "db_engine_version": "5.5",
   "db_flavor_id": "db.m1.small",
   "db_allocated_storage": "12",
   "ec2_key_name": "genigames",
-  "ec2_image_id": "ami-0721ac6e",
-  "ec2_flavor_id": "c1.medium"
+  "ec2_image_id": "ami-61e51f0a",
```

Comment:
This ami reference `ami-0721ac6e` is to the new one you created with `script/update_base_ruby_box.rb` right?